subsys/logging/Kconfig.backends

```
@@ -3,20 +3,56 @@
 
 menu "Backends"
 
-config LOG_BACKEND_UART
+menuconfig LOG_BACKEND_UART
```

Using a menuconfig config here  will give you an extra level when having to change backend.

First entering the `menu` (that only has a single choice), and thereafter entering the choice selection itself which is an additional layer.

The choice itself creates a menu for selection, and is only visible when `LOG_BACKEND_UART=y`.

```suggestion
config LOG_BACKEND_UART
```